.github/workflows/alpha-build-and-deploy.yml

```diff
@@ -23,14 +23,16 @@ jobs:
       # This step will configure environment variables to be used by all steps
       # involving AWS interaction further down
       - name: AWS credentials configuration
-        uses: aws-actions/configure-aws-credentials@v2
+        uses: aws-actions/configure-aws-credentials@v4
         with:
           role-to-assume: ${{secrets.GH_ACTIONS_AWS_ROLE}}
           role-session-name: gh-actions-${{github.run_id}}.${{github.run_number}}.${{github.run_attempt}}-build-image
           aws-region: us-east-1
       - name: Amazon ECR login
         id: login-ecr
-        uses: aws-actions/amazon-ecr-login@v1
+        uses: aws-actions/amazon-ecr-login@v2
+        with:
+          mask-password: 'true'
```

Comment:
@abecerra `mask-password` defaults to true in this aws-actions/amazon-ecr-login release, so you can remove the `with:` section here.